help/CreateImageAndAzureResources.md

```diff
@@ -1,11 +1,122 @@
-# Build an image.
+### General
```

Comment:
I think we should change header structure. The root header should be `#`. Nested +1

Reply:
Yep.